From 8c23c71cd1f7f68e06e37af5491cab5dba4ec970 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Thu, 14 Feb 2013 14:25:00 +0100
Subject: [PATCH] Remove outdated mention of dselect documentation

There's no description in the appendices of the dselect access methods.
---
 policy.sgml | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 73b8d6a..ddf7698 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -10838,12 +10838,6 @@ END-INFO-DIR-ENTRY
 	they interact with packages.</p>
 
       <p>
-	It also documents the interaction between
-	<prgn>dselect</prgn>'s core and the access method scripts it
-	uses to actually install the selected packages, and describes
-	how to create a new access method.</p>
-
-      <p>
 	This manual does not go into detail about the options and
 	usage of the package building and installation tools.  It
 	should therefore be read in conjunction with those programs'
-- 
1.8.1.3

